packages/flutter/lib/src/material/app.dart

```diff
@@ -52,6 +52,7 @@ class MaterialApp extends StatefulWidget {
     this.theme,
     this.home,
     this.routes: const <String, WidgetBuilder>{},
+    this.initialRoute,
```

Comment:
The Navigator's properties aren't just passed through. For example, onGenerateRoute is hooked by MaterialApp, key is hooked by WidgetsApp.